pywinauto/controls/uia_controls.py

```diff
@@ -131,3 +135,101 @@ def is_selected(self):
         elem = self.element_info.element
         iface = uia_defs.get_elem_interface(elem, "SelectionItem")
         return iface.CurrentIsSelected
+
+#====================================================================
+class ComboBoxWrapper(UIAWrapper.UIAWrapper):
+    """Wrap a UIA CoboBox control"""
```

Comment:
(minor) Landscape.io recommends to have an empty line before a class doc string and after it.